src/Database/Schema/PostgresSchemaDialect.php

```diff
@@ -67,7 +67,20 @@ public function listTablesWithoutViewsSql(array $config): array
      */
     public function describeColumnSql(string $tableName, array $config): array
     {
-        $sql = 'SELECT DISTINCT table_schema AS schema,
+        $sql = $this->describeColumnQuery();
+        $schema = empty($config['schema']) ? 'public' : $config['schema'];
```

Comment:
```suggestion
        $schema = $config['schema']) ?? 'public';
```
Should we just reduce this to standard coalescence?